lib/disk/modules/RawBlockIO.rb

```diff
@@ -1,14 +1,10 @@
-
-$:.push("#{File.dirname(__FILE__)}")
-
-require 'io/extra'
-require 'MiqBlockDevOps'
+require_relative 'MiqBlockDevOps'
 
 class RawBlockIO
 
   MIN_SECTORS_TO_CACHE = 64
 
-  def initialize(filename, mode="r")
+  def initialize(filename, mode=File::RDONLY|File::DIRECT)
```

Comment:
@djberg96 There's a problem doing it this way. If the client passes in a `mode`, then it will replace the flags you've set here. The File::DIRECT flag needs to be added to whatever `mode` the client passes in - on the `open` call. However, I'm not sure if there's a clean way of accounting for the possibility of a combination of character and bit-bask flags.